src/components/Link/Link.module.css

```diff
@@ -51,22 +55,27 @@
    */
   &.link--size-xl {
     font: var(--eds-theme-typography-body-xl);
+    font-weight: 500;
   }
 
   &.link--size-lg {
     font: var(--eds-theme-typography-body-lg);
+    font-weight: 500;
   }
 
   &.link--size-md {
     font: var(--eds-theme-typography-body-md);
+    font-weight: 500;
   }
 
   &.link--size-sm {
     font: var(--eds-theme-typography-body-sm);
+    font-weight: 500;
   }
 
   &.link--size-xs {
     font: var(--eds-theme-typography-body-xs);
+    font-weight: 500;
```

Comment:
Oh just so 500 is the expected font size for Links?

Reply:
500 is the weight for each link size. We have to add them here b/c `font:` will try to set it to 400 which is what we saw. later this year when we revisit the typography tokens, we won't need this anymore (will be transparent to you all when that happens)